python/cudf/cudf/tests/test_orc.py

```diff
@@ -738,3 +738,19 @@ def test_nanoseconds_overflow():
 
     pyarrow_got = pa.orc.ORCFile(buffer).read()
     assert_eq(expected.to_pandas(), pyarrow_got.to_pandas())
+
+
+def test_empty_dataframe():
+    buffer = BytesIO()
+    expected = cudf.DataFrame()
+    expected.to_orc(buffer)
+
+    # Raise error if column name is mentioned, but it doesn't exist.
```

Comment:
so the error user gets here will be "Filtered out all columns"?

Reply:
Yes

Reply:
Shouldn't it be an error to ask for selecting a column that doesn't exist.